### DISCUSSION TITLE
How to get the two block at the same size

### DISCUSSION THREAD

**Discussion by ArcadeCode** [anuraghazra/github-readme-stats#3699] [Q&A] [answered]
Okay so I have this code : 
```html
           <div style="display:flex; align-items:center;">
                <a href="https://github.com/ArcadeCode" target="_blank">
                    <img id="targetImgForGetSize" src="https://github-readme-stats.vercel.app/api?username=ArcadeCode&show_icons=true&theme=material-palenight&count_private=true&hide_border=true" />
                </a>
                <a href="https://github.com/ArcadeCode" target="_blank">
                    <img id="targetImgToSetSize" src="https://github-readme-stats.vercel.app/api/top-langs?username=ArcadeCode&show_icons=true&theme=material-palenight&hide_border=true&layout=compact" />
                </a>  
            </div>
```
I want to have theses two div at the same size, I tried this : 
```javascript
var img = document.getElementById("targetImgForGetSize"); 
var width = img.clientWidth;
var height = img.clientHeight;
    
var resizeImg = document.getElementById("targetImgForGetSize")
resizeImg.style.height = height+" px"
resizeImg.style.width = width+" px"
```
But this don't work, I have search but there is no size indicator on the doc right ?

Thanks if you help me ^^

**Comment by qwerty541** [comment, marked as answer]
Hey, @ArcadeCode! Is that the result which you want to reach? 

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/8b1139de-5f0b-4e92-b51e-6acb2821bd90)

In case if it is, then here is a code:

```html
<a href="https://github.com/ArcadeCode">
  <img height=190 align="center" src="https://github-readme-stats.vercel.app/api?username=ArcadeCode&show_icons=true&theme=material-palenight&count_private=true&hide_border=true" />
</a>
<a href="https://github.com/ArcadeCode">
  <img height=190 align="center" src="https://github-readme-stats.vercel.app/api/top-langs?username=ArcadeCode&show_icons=true&theme=material-palenight&hide_border=true&layout=compact" />
</a>
```

**Reply by ArcadeCode** [reply]
It was so simple -_- thank you to answer my problem 😅

**Reply by qwerty541** [reply]
Glad to help! Feel free to contact me if you need any further assistance.

**Reply by qwerty541** [reply]
Hey, @ArcadeCode. I see that you copied the code into your profile readme, but looks like the size of discussion comment area is different to the size of profile readme area, as you can see on attached screenshot this difference cause line break. You can descrease value of height attributes in order to fix this.

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/d20c82ab-ed60-4cfb-8b26-f62c9e25839b)

**Reply by ArcadeCode** [reply]
Oh yes I will do it later thank 😁
